From 88bea067f8df836909a8dec32d79a6e9a5779ad8 Mon Sep 17 00:00:00 2001
From: Derek Higgins <derekh@redhat.com>
Date: Wed, 26 Jun 2013 14:39:17 +0100
Subject: [PATCH] Temp Fedora incuabtor patch

---
 scripts/install-dependencies | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/scripts/install-dependencies b/scripts/install-dependencies
index e502830..dd9cc61 100755
--- a/scripts/install-dependencies
+++ b/scripts/install-dependencies
@@ -41,9 +41,10 @@ fi
 
 if [ "$os" = "redhat" ]; then
   # For RHEL/CentOS, python-pip is in EPEL
-  sudo yum install -y python-lxml libvirt-python libvirt qemu-img qemu-kvm git python-pip openssl-devel python-devel gcc audit python-virtualenv
+  sudo yum install -y python-lxml libvirt-python libvirt qemu-img qemu-kvm git python-pip openssl-devel python-devel gcc audit python-virtualenv openvswitch
 
   sudo service libvirtd restart
+  sudo service openvswitch restart
 fi
 
 $BASE/scripts/setup-clienttools
-- 
1.8.1.4

