From 28fc030839d14a6e6b5a6b0e1ecce3b49f2cc37e Mon Sep 17 00:00:00 2001
From: Derek Higgins <derekh@redhat.com>
Date: Wed, 15 May 2013 12:01:46 +0100
Subject: [PATCH] Create a ssh key-pair if non exists

---
 scripts/install-dependencies | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/install-dependencies b/scripts/install-dependencies
index 354da1d..e35c65e 100755
--- a/scripts/install-dependencies
+++ b/scripts/install-dependencies
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -eu
 
+if [ ! -e ~/.ssh/id_rsa.pub ] ; then
+  ssh-keygen -N '' -f ~/.ssh/id_rsa
+fi
+
 # keys
 if ! grep "$(cat ~/.ssh/id_rsa.pub)" ~/.ssh/authorized_keys >/dev/null; then
   cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
-- 
1.8.1.4

