From 02818225efe2574f2e185ab7c023f2a10e755805 Mon Sep 17 00:00:00 2001
From: Derek Higgins <derekh@redhat.com>
Date: Wed, 3 Jul 2013 12:56:22 +0100
Subject: [PATCH] No --prealloc-metadata on fedora's virsh

---
 scripts/create-nodes | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/create-nodes b/scripts/create-nodes
index cb3d94c..0bedb5b 100755
--- a/scripts/create-nodes
+++ b/scripts/create-nodes
@@ -13,6 +13,6 @@ count=0
 
 for (( count=0; count<TOTAL; ++count )) ; do
   # TODO: make this idempotent / at least not error when run more than once.
-  virsh vol-create-as default baremetal-$count.qcow2 ${DISK}G --format qcow2 --prealloc-metadata
+  virsh vol-create-as default baremetal-$count.qcow2 ${DISK}G --format qcow2 #--prealloc-metadata
   configure-vm --bootdev network --name baremetal_$count --image /var/lib/libvirt/images/baremetal-$count.qcow2 --arch i686 --cpus $CPU --memory $MEM
 done
-- 
1.8.1.4

